api/PluginArray.json

```diff
@@ -23,10 +23,10 @@
             "version_added": null
           },
           "opera": {
-            "version_added": true
+            "version_added": "15"
```

Comment:
I wonder if this wasn't supported in Opera Presto. `PluginArray` is the type for `navigator.plugins`, can you update that entry at the same time and see if the values are the same? I suspect that the interface object just wasn't exposed, and that this needs to test an instance, which is simply `navigator.plugins`.

Reply:
Looks like this is a case of unexposed interface, you're right.  Adding a custom test, and manually updated the data.